src/core/linalg/src/sparse/4C_linalg_sparsematrixbase.hpp

```diff
@@ -138,7 +138,7 @@ namespace Core::LinAlg
     {
       if (!row_map_)
       {  // check if view is uninitialized
-        row_map_ = Core::LinAlg::Map(sysmat_->RowMap());
+        row_map_.emplace(sysmat_->RowMap());
```

Comment:
@maxfirmbach Note that this PR will _not_ fix the problems we get when somebody replaces maps (at least I think so). This needs to be further isolated.